test/model/game_state_test.dart

```diff
@@ -115,5 +117,48 @@ void main() {
     });
   });
 
+  group('Special cards', ()
```

Comment:
Would be nice to test the negative cases, e.g. a hand with 14 cards, one food and just under the Infamy threshold should draw the next card like normal?

Reply:
Agreed, done.